server/mcp_server_eip/README.md

```diff
@@ -0,0 +1,104 @@
+# EIP MCP Server 
+
+## 版本信息
+v1.0
+
+## 产品描述
+
+EIP MCP Server 是一个模型上下文协议(Model Context Protocol)服务器，为MCP客户端(如Claude Desktop)提供与火山引擎EIP服务交互的能力。可以基于自然语言对云端资源进行管理，支持查询公网IP、带宽包实例等操作。
+## 分类
+网络
+
+## 功能
+
+- 查看一个指定公网IP的详细信息
+- 查询满足指定条件的公网IP
+- 查询满足指定条件的共享带宽包
+
+## Tools
+本 MCP Server 产品提供以下 Tools (工具/能力):
+
+业务数据查询
+- `DescribeEipAddressAttributes`: [查看指定公网IP的详情](https://www.volcengine.com/docs/6402/71040)
```

Comment:
it is not consistent with tools name and description describe_eip_addresses in mcp server.